packages/mds-utils/state-machine.ts

```diff
@@ -63,7 +141,7 @@ const generateGraph = () => {
     const eventTransitions: VEHICLE_EVENT[] = Object.keys(stateTransitionDict[status]) as VEHICLE_EVENT[]
     for (const event of eventTransitions) {
       if (event) {
-        const nextStatus: VEHICLE_STATE | undefined = stateTransitionDict[status][event]
+        const nextStatus: Array<VEHICLE_STATE> | undefined = stateTransitionDict[status][event]
```

Comment:
Are we using `generateGraph()` @karcass? If not we can probably remove this code.

Reply:
Nah let's keep it. I'm using it to generate the new state transition diagram.

Reply:
I'd like to 👀 the files, are you using GraphViz?